app/components/tutorial_highlight/index.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modal, StyleSheet, useWindowDimensions, View} from 'react-native';
+
+import HighlightItem from './item';
+
+type Props = {
+    children?: React.ReactNode;
+    itemBounds: TutorialItemBounds;
+    itemBorderRadius?: number;
+    onDismiss: () => void;
+    onShow: () => void;
+}
+
+const TutorialHighlight = ({children, itemBounds, itemBorderRadius, onDismiss, onShow}: Props) => {
+    const {width, height} = useWindowDimensions();
+    const isLandscape = width > height;
+    const supportedOrientations = isLandscape ? 'landscape' : 'portrait';
```

Comment:
should we add a check if it's tablet also ?

Reply:
there is no need as for phones device is lock to portrait.